package.json

```diff
@@ -100,25 +100,24 @@
     "elasticsearch": "^14.2.1",
     "elasticsearch-browser": "^14.2.1",
     "encode-uri-query": "1.0.0",
-    "even-better": "7.0.2",
     "execa": "^0.10.0",
     "expiry-js": "0.1.7",
     "extract-text-webpack-plugin": "3.0.1",
     "file-loader": "1.1.4",
     "font-awesome": "4.4.0",
     "glob": "^7.1.2",
     "glob-all": "^3.1.0",
+    "good": "github:elastic/good#v8.1.1-kibana1",
```

Comment:
I know I suggested pulling in the dependency like this, but this won't work after all. Any dependency that is referenced via a github link/url (or any url) will cause yarn to reach out and resolve those packages every time. I guess the thought process is that yarn can't reliably determine the state of a package unless it can rely on the behaviors of the upstream service, and it hasn't built-in knowledge of github commit handling like it has npm.

Looks like we'll need to publish this to `@elastic/good` on npm instead. Ping me directly and we can chat details.